client/src/components/classesPage/forms/RecipientForm.js

```diff
@@ -10,36 +10,39 @@ const styles = theme => ({
 });
 
 function RecipientForm(props) {
+  const { classlist } = props;
+
   const [recipient, setRecipient] = useState({
+    id: '',
     email: '',
-    first_name: '',
-    last_name: ''
+    firstname: '',
+    lastname: ''
   });
 
-  const handleChange = e => {
-    e.preventDefault();
+  const handleChange = ({ target: { name, value } }) => {
     setRecipient({
       ...recipient,
-      [e.target.name]: e.target.value
+      [name]: value
     });
   };
 
   const handleSubmit = e => {
     e.preventDefault();
     const new_recipient = {
+      id: classlist.length + 2,
       email: recipient.email,
-      first_name: recipient.first_name,
-      last_name: recipient.last_name
+      firstname: recipient.firstname,
+      lastname: recipient.lastname
```

Comment:
I think this will bring us some errors with Sendgrid if saved to the parent component. I believe keeping `first_name` and `last_name` is necessary.

This is the doc section: https://sendgrid.api-docs.io/v3.0/contacts-api-recipients/add-recipients